rskj-core/src/main/java/org/ethereum/net/client/Capability.java

```diff
@@ -65,6 +67,11 @@ public boolean equals(Object obj) {
         }
     }
 
+    @Override
+    public int hashCode() {
+        return Objects.hash(name, version);
```

Comment:
not related to the change itself, but as a sort of improvement, I'd mark `name` and `version` fields as `final` ones to explicitly define `Capability` as an immutable class.

Reply:
Done!